addons/purchase/views/purchase_views.xml

```diff
@@ -348,7 +349,7 @@
                                                              <span>
                                                                  Quantity:
                                                                  <t t-esc="record.product_qty.value"/>
-                                                                 <t t-esc="record.product_uom.value" groups="uom.group_uom"/>
+                                                                 <t t-esc="' ' + record.product_uom.value" groups="uom.group_uom"/>
```

Comment:
It's quite hacky to put a space within the `t-esc` like this. Can you instead rearrange the xml so that the space appears? Kind of like it's done here:

https://github.com/odoo/odoo/blob/f168c4ba9d64a05a4a55715c44016a15230fe706/addons/mrp/views/mrp_production_views.xml#L532

![image](https://github.com/odoo/odoo/assets/57942143/0a06fdf1-df50-4f76-8ec1-5c8aacf96156)